coaddAnalysis.py

```diff
@@ -0,0 +1,234 @@
+    pad : `float`
+       Pad the boundary by pad degrees
+    nDecimals : `int`
+       Round coordinates to this number of decimal places
+    raMin, raMax : `float`
+       Initiate minimum[maximum] RA determination at raMin[raMax] (deg)
+    decMin, decMax : `float`
+       Initiate minimum[maximum] DEC determination at decMin[decMax] (deg)
+    Returns
+    -------
+    `lsst.pipe.base.Struct`
+       Contains the ra and dec min and max values for the patchList provided
+    """
+    for ip, patch in enumerate(tractInfo):
+        if str(patch.getIndex()[0])+","+str(patch.getIndex()[1]) in patchList:
+            raPatch, decPatch = bboxToRaDec(patch.getOuterBBox(), tractInfo.getWcs())
+            raMin = min(np.round(min(raPatch) - pad, nDecimals), raMin)
+            raMax = max(np.round(max(raPatch) + pad, nDecimals), raMax)
+            decMin = min(np.round(min(decPatch) - pad, nDecimals), decMin)
+            decMax = max(np.round(max(decPatch) + pad, nDecimals), decMax)
+    return Struct(
+        raMin=raMin,
+        raMax=raMax,
+        decMin=decMin,
+        decMax=decMax,
+    )
+
+
+def plotTractOutline(tractInfo, patchList, axes=None, fontSize=5, maxDegBeyondPatch=1.5):
+    """Plot the the outline of the tract and patches highlighting those with data
+    As some skyMap settings can define tracts with a large number of patches, this can
+    become very crowded.  So, if only a subset of patches are included, find the outer
+    boudary of all patches in patchList and only plot to maxDegBeyondPatch degrees
+    beyond those boundaries (in all four directions).
+    Parameters
+    ----------
+    tractInfo : `lsst.skymap.tractInfo.ExplicitTractInfo`
+       Tract information object for extracting tract RA and DEC limits.
+    patchList : `list` of `str`
+       List of patch IDs with data to be plotted.  These will be color shaded in the outline plot.
+    fontSize : `int`
+       Font size for plot labels.
+    maxDegBeyondPatch : `float`
+       Maximum number of degrees to plot beyond the border defined by all patches with data to be plotted.
+    """
+    if axes is None:
+        fig = plt.figure(figsize=(10, 10))
+        axes = fig.gca()
+    buff = 0.02
+    axes.tick_params(which="both", direction="in", labelsize=fontSize)
+    axes.locator_params(nbins=6)
+    axes.ticklabel_format(useOffset=False)
+
+    tractRa, tractDec = bboxToRaDec(tractInfo.getBBox(), tractInfo.getWcs())
+    patchBoundary = getRaDecMinMaxPatchList(patchList, tractInfo, pad=maxDegBeyondPatch)
+
+    xMin = min(max(tractRa), patchBoundary.raMax) + buff
+    xMax = max(min(tractRa), patchBoundary.raMin) - buff
+    yMin = max(min(tractDec), patchBoundary.decMin) - buff
+    yMax = min(max(tractDec), patchBoundary.decMax) + buff
+    xlim = xMin, xMax
+    ylim = yMin, yMax
+    axes.fill(tractRa, tractDec, fill=True, edgecolor='k', lw=1, linestyle='solid',
+              color="black", alpha=0.2)
```

Comment:
Belated review comment - there was a deprecation warning about calling edgecolor before color in one of your notebooks. Please address this by switching the argument order here.